superset/assets/src/components/ObjectTags.less

```diff
@@ -0,0 +1,213 @@
+/**
```

Comment:
Is it necessary to be a `less` file? From quick skimming, a `*.css` extension should be enough.